### PR TITLE
OGD-420 Publish API individuals-matching-api

### PR DESCRIPTION
**Pull request by edgarjimenez**

### CODE REVIEW

File: resources/public/api/conf/1/examples/matched-response.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "_links": [
+    {
+      "name": "self",
+      "href": "/customer/matched/633e0ee7-315b-49e6-baed-d79c3dffe467"
```
Comment (tom59) [inline resources/public/api/conf/1/examples/matched-response.json]: /individuals/matching/633e0ee7-315b-49e6-baed-d79c3dffe467

File: resources/public/api/conf/1/sandbox/match-individual.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                                                  "message": "This field is required",<br>
+                                                  "path": "/firstName"<br>
+                                                }<br>
+                                            ]<br>
+                                         }
+                </p>
+            </td>
+        </tr>
+        <tr>
+        	 <td><p>Request with a valid payload, but the customer could not be found</p></td>
```
Comment (tom59) [inline resources/public/api/conf/1/sandbox/match-individual.md]: individual

File: app/uk/gov/hmrc/individualsmatchingapi/controllers/DocumentationController.scala [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import play.api.Configuration
+import play.api.http.HttpErrorHandler
+import play.api.mvc.{Action, AnyContent}
+import uk.gov.hmrc.individualsmatchingapi.views._
+
+
+@Singleton
+class DocumentationController @Inject()(httpErrorHandler: HttpErrorHandler, configuration: Configuration) extends uk.gov.hmrc.api.controllers.DocumentationController(httpErrorHandler) {
+
+  private lazy val whitelistedApplicationIds = configuration.getStringSeq("api.access.whitelistedApplicationIds").getOrElse(Seq.empty)
```
Comment (tom59) [inline app/uk/gov/hmrc/individualsmatchingapi/controllers/DocumentationController.scala]: can we call it api.access.version-P1.whitelistedApplicationIds to be consistent and make it clear it is for privileged?

File: resources/public/api/conf/1/examples/matched-response.json (hunk 1 above)
Comment (edgarjimenez) [reply resources/public/api/conf/1/examples/matched-response.json]: 👍

File: resources/public/api/conf/1/sandbox/match-individual.md (hunk 2 above)
Comment (edgarjimenez) [reply resources/public/api/conf/1/sandbox/match-individual.md]: 👍

File: app/uk/gov/hmrc/individualsmatchingapi/controllers/DocumentationController.scala (hunk 3 above)
Comment (edgarjimenez) [reply app/uk/gov/hmrc/individualsmatchingapi/controllers/DocumentationController.scala]: 👍

**Pull request merged**